libs/openFrameworks/graphics/ofTrueTypeFont.h

```diff
@@ -377,6 +377,13 @@ class ofTrueTypeFont{
     /// \returns current font direction
 	void setDirection(ofTrueTypeFontDirection direction);
 
+	float charWidth(uint32_t c) const{
+		return getGlyphProperties(c).width;
+	}
+	float charAdvance(uint32_t c) const{
+		return getGlyphProperties(c).advance;
+	}
+
```

Comment:
We should have this in the cpp and maybe with better naming. 
`getCharWidth` `getCharAdvance` ?

Reply:
Sure! Other idea we can just make getGlyphProperties public, so this functions are not needed

Reply:
I've just renamed for now, I can separate to cpp soon